src/t_stream.c

```diff
@@ -2712,10 +3089,15 @@ void xdelCommand(client *c) {
  *                             the specified length. Use ~ before the
  *                             count in order to demand approximated trimming
  *                             (like XADD MAXLEN option).
+ * MINID [~|=] <id>         -- Trim so that the stream will not contain entries
+ *                             with IDs smaller than 'id'. Use ~ before the
+ *                             count in order to demand approximated trimming
+ *                             (like XADD MINID option).
+ * MAXAGE [~|=] <ms>        -- Trim so that the stream will not contain entries
+ *                             with IDs older than (now-'ms'). Use ~ before the
+ *                             count in order to demand approximated trimming
+ *                             (like XADD MAXAGE option).
```

Comment:
Also add MAXNODES here?

I assume MAXNODES refers to the number of radix tree nodes. Approximately how many entries are there per node?

Btw, adding a default value of 100 for MAXNODES affects the legacy MAXLEN option, so even if `=` is given, MAXLEN will not enforce the maximum length of the stream anymore. (Instead MAXLEN actually specifies the *minimum length to keep* when trimming.) I think the change is to the better since it limits the execution time in corner cases, but I think it needs to be documented.

The docs for MAXAGE and MINID should also have some note saying something like that the limit is not enforced (for performance reasons or whatever, even if MAXNODES isn't documented) even when `=` is given, but that the actual guarantee is that entries back to the given age or ID are kept when trimming.